lte/gateway/python/magma/mobilityd/uplink_gw.py

```diff
@@ -57,7 +57,7 @@ def __init__(self, gw_info_map: MutableMapping[str, GWInfo]):
         self._backing_map = gw_info_map
         self._read_default_gw_timer = None
         self._read_default_gw_timer6 = None
-        self._read_default_gw_interval_seconds = 20
+        self._read_default_gw_interval_seconds = 5
 
```

Comment:
In magma vagrant VM, the IPv4 and v6 gateway is configured, so this timer is never used, why does this frequency matter ?